packages/app/src/systems/Settings/machines/settingsMachine.ts

```diff
@@ -0,0 +1,150 @@
+import type { InterpreterFrom, StateFrom } from 'xstate';
+import { assign, createMachine } from 'xstate';
+
+import type { AccountInputs } from '~/systems/Account';
+import { AccountService } from '~/systems/Account';
+import type { ChildrenMachine } from '~/systems/Core';
+import { FetchMachine } from '~/systems/Core';
+import type { UnlockMachine } from '~/systems/DApp';
+
+type MachineServices = {
+  getMnemonicPhrase: {
+    data: string[];
+  };
+};
+
+type MachineContext = {
+  words: string[];
+};
+
+type MachineEvents = (
+  | { type: 'UNLOCK_WALLET'; input: AccountInputs['unlock'] }
+  | { type: 'CHANGE_PASSWORD'; input: AccountInputs['changePassword'] }
+) & { data?: string[] };
+
+export const settingsMachine = createMachine(
+  {
+    // eslint-disable-next-line @typescript-eslint/consistent-type-imports
+    tsTypes: {} as import('./settingsMachine.typegen').Typegen0,
+    schema: {
+      context: {} as MachineContext,
+      services: {} as MachineServices,
+      events: {} as MachineEvents,
+    },
+    predictableActionArguments: true,
+    id: '(machine)',
+    initial: 'unlocking',
```

Comment:
initial should really be `unlocking`? feels wrong from the flow I'm imagining as you don't start nothing by `unlocking`